packages/list/mwc-list-base.ts

```diff
@@ -29,11 +29,30 @@ import {Layoutable, ListItemBase, RequestSelectedDetail} from './mwc-list-item-b
 
 export {ActionDetail, createSetFromIndex, isEventMulti, isIndexSet, MWCListIndex, SelectedDetail} from './mwc-list-foundation';
 
+function debounceLayout(
+    callback: <T>(this: T, updateItems: boolean) => void, waitInMS = 50) {
+  let timeoutId: NodeJS.Timeout;
```

Comment:
This type is causing crashes internally. Could it be possible to cast it as `number` and you might also have to cast `clearTimeout` as `(id: number) => void`?